x11-misc/polybar/polybar-3.3.1.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2019 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
Looking at eclasses that are being used in here, EAPI-7 should work. Can you please test and update it?
Also for consistency there's usually a blank line below EAPI.

Reply:
Added a blank line.

Setting `EAPI=7` results in linker errors (undefined reference to symbols `xcb_icccm*` and `xcb_ewmh*`).

Reply:
Ahh, could be because with EAPI-7 cmake-utils puts BUILD_SHARED_LIBS on by default. 

I couldn't build this, but I couldn't build 3.2.1 either so it's probably something on my end. Ebuild looks good to me, let's hope @johu picks it up soon and tests it.